### ISSUE TITLE
[C++] Published new library panda-apache

### ISSUE THREAD

**Issue by ava6969** [apache/arrow#33747]
### Describe the enhancement requested

This library creates a pandas interface over arrow Apache. It still maintains Apache performance. If it will be useful to you. I am open to more collaboration.

https://github.com/ava6969/panda-arrow.git

### Component(s)

C++

**Comment by kou** [comment]
Are you proposing importing your project to apache/arrow?
If so, could you start a discussion on `dev@arrow.apache.org` mailing list? https://arrow.apache.org/community/

FYI: We have a discussion for data frame API: https://lists.apache.org/thread/50vbmw49w83sj3km326srown64c7hlf1

**Comment by ava6969** [comment]
that would be great if it could be integrated as i had to copy some internal files.

**Comment by ava6969** [comment]
how does mailing list discussion work, do i email dev@arrow.apache.org

**Comment by kou** [comment]
Yes. But you need to subscribe the mailing list before you send an e-mail. See https://arrow.apache.org/community/ how to subscribe.

**Comment by ava6969** [comment]
thanks just did

**Comment by thisisnic** [comment]
This issue has been marked as stale because it has had no activity in the past 2 years. Please remove the stale label or comment below, or this issue will be closed in 14 days.
